### ISSUE TITLE
The application crashes due to wrong return.

### ISSUE THREAD

**Issue by fogfish** [virtan/eep#12]
The application crashes due to wrong return.
```erlang
(eep@127.0.0.1)1> application:start(eep).

=INFO REPORT==== 19-May-2016::11:43:47 ===
    application: eep
    exited: {bad_return,{{eep_app,start,[normal,[]]},ok}}
    type: temporary
{error,{bad_return,{{eep_app,start,[normal,[]]},ok}}}
```
Inclusion of application into release with rebar3 terminates the node
```erlang
(typhoon@127.0.0.1)1> {"Kernel pid terminated",application_controller,"{application_start_failure,eep,{bad_return,{{eep_app,start,[normal,[]]},ok}}}"}
Kernel pid terminated (application_controller) ({application_start_failure,eep,{bad_return,{{eep_app,start,[normal,[]]},ok}}})
heart: Thu May 19 11:40:00 2016: Erlang is crashing .. (waiting for crash dump file)
heart: Thu May 19 11:40:00 2016: Would reboot. Terminating.
```

**Comment by fogfish** [comment]
The fix available at #11

**Comment by virtan** [comment]
Thanks. Applied.
